com.unity.shadergraph/Editor/Data/Nodes/NodeClassCache.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Profiling;
+
+namespace UnityEditor.ShaderGraph
+{
+    [InitializeOnLoad]
+    public static class NodeClassCache
```

Comment:
Is this intended to be public?

Reply:
same as previous comment, will make internal